### PR TITLE
V2

### PR DESCRIPTION
**Pull request by xrotwang**
The Python API should be backwards compatible, the cli has been slightly changed, i.e. commands may have a different name now, or different options. Overall, the commands are a lot more homogeneous now, e.g. in the way output is specified.
In fact, we nw could also try to make commands read nexus from stdin - and make the commands ready for chaining.

first stab at #13

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,8 @@
 
 A Generic nexus (.nex, .trees) reader/writer for python.
 
-[![Build Status](https://travis-ci.org/SimonGreenhill/python-nexus.svg?branch=master)](https://travis-ci.org/SimonGreenhill/python-nexus)
-[![Coverage Status](https://coveralls.io/repos/SimonGreenhill/python-nexus/badge.svg?branch=master&service=github)](https://coveralls.io/github/SimonGreenhill/python-nexus?branch=master)
+[![Build Status](https://travis-ci.org/shh-dlce/python-nexus.svg?branch=master)](https://travis-ci.org/shh-dlce/python-nexus)
+[![codecov](https://codecov.io/gh/shh-dlce/python-nexus/branch/master/graph/badge.svg)](https://codecov.io/gh/shh-dlce/python-nexus)
 [![DOI](https://zenodo.org/badge/22704/SimonGreenhill/python-nexus.svg)](https://zenodo.org/badge/latestdoi/22704/SimonGreenhill/python-nexus)
 
 Copyright (c) 2009-2019, Simon J. Greenhill <simon@simon.net.nz>
```
Comment (SimonGreenhill) [inline README.md]: add your name here too, or remove the copyright line

**Review by SimonGreenhill — APPROVED** [review]
Looks good!

**Comment by SimonGreenhill** [comment]
Awesome, thanks @xrotwang! looks like a well-needed tidyup.

File: README.md (hunk 1 above)
Comment (xrotwang) [reply README.md]: will. and could you add me as maintainer/owner on pypi, too?

Comment (SimonGreenhill) [reply README.md]: sure -- that should be done now.

**Comment by SimonGreenhill** [comment]
& I like the idea about chaining stdin... but that might be a new issue.

**Comment by xrotwang** [comment]
And one more question: What about re-licensing under Apache 2?

**Comment by SimonGreenhill** [comment]
No strong feelings either way, aren't BSD-3 and Apache 2 compatible with the only differences being due to patenting?

**Comment by xrotwang** [comment]
Ah yes, BSD 2 is fine for me, too. Just the current content of LICENSE looked like it was neither.

**Comment by SimonGreenhill** [comment]
I think it's supposed to be BSD 3, so how about we convert to BSD-2 (I'm not worried that someone will mis-use the good name of 'python-nexus' so that 3rd clause is moot anyway)?

**Review by SimonGreenhill — APPROVED** [review]

**Pull request merged**